app/components/youtube/client.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const youtubeVideoPlayer = require('../../services/universal/youtube-video-player'),
+  { Visible, isElementNotHidden } = require('../../services/client/visibility');
+
+module.exports = (el) => {
+  let autoplay = el.getAttribute('data-autoplay-video') === 'true',
+    videoConfig = {
+      videoContainerId: el.getAttribute('data-element-id').trim(),
+      videoId: el.getAttribute('data-video-id').trim(),
+      // player variables and settings
+      playerParams: {
+        loop: 1,
+        listType: 'playlist',
+        list: el.getAttribute('data-playlist').trim(),
+        autoplay: autoplay ? 1 : 0,
+        controls: 1,
+        enablejsapi: 1,
+        modestbranding: 1,
+        rel: 0,
+        showinfo: 0,
+        wmode: 'transparent'
+      },
+      customParams: {
+        autoPlayNextVideo: el.getAttribute('data-autoplay-next-video').trim(),
+        trackVideoType: el.getAttribute('data-track-video-type').trim(),
+        customPlayer: el.getAttribute('data-custom-play').trim(),
+        templateid: el.getAttribute('data-element-id').trim(),
+        muted: autoplay // always mute autplaying videos
+      }
+    },
+    visible;
```

Comment:
And declare just this one as `let`
```suggestion
  let visible;
```